libs/blocks/gnav/gnav.js

```diff
@@ -413,8 +436,11 @@ class Gnav {
       const ioResp = await fetch(`https://${env.adobeIO}/profile`, { headers: new Headers({ Authorization: `Bearer ${accessToken.token}` }) });
 
       if (ioResp.status === 200) {
+        const gnav = profileEl.closest('.gnav');
+        gnav.classList.add('signed-in');
```

Comment:
Is this necessary?

Reply:
I use this to change the css-grid layout to allow the app launcher icon to be added after the profile and before the ending logo icon - since this does not exists with out being signed in.